tools/glyph-studio/py/typography_runs.py

```
@@ -0,0 +1,796 @@
+    from receipt_line_scorecard import _load_words_and_real
+
+    from glyph_segment import auto_polarity, sauvola_mask
+
+    real, words = _load_words_and_real(merchant, image_id, receipt_id)
+    overlaps = ocr_overlap_score(words)
+    gray = np.asarray(real.convert("L"))
+    H, W = gray.shape
+    details = client.get_image_details(image_id)
+    letters = [
+        l
+        for l in details.receipt_letters
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)